plugins/modules/web_infrastructure/jira.py

```diff
@@ -106,6 +108,20 @@
        (possibly after merging with other required data, as when passed to create). See examples for more information,
        and the JIRA REST API for the structure required for various fields.
 
+  jql:
+    required: false
+    version_added: '2.10'
+    description:
+     - Query JIRA in JQL Syntax, e.g. 'CMDB Hostname'='test.example.com'.
+    type: str
+
+  maxresults:
+    required: false
+    version_added: '2.10'
+    description:
+     - Limit the max number of returned resuts from a jira search. If not value is specified, the default jira limit will be used.
```

Comment:
```suggestion
     - Limit the result of I(operation=search). If no value is specified, the default jira limit will be used.
     - Used when I(operation=search) only, ignored otherwise.
```
Just a suggestion
anyway, there is a typo in ```resuts```  in the origin sentence